### PR TITLE
Remove client duplications

### PR DESCRIPTION
**Pull request by roszpun**

### CODE REVIEW

File: src/js/clients.js [hunk 1]
```diff
@@ -724,7 +708,7 @@ if (document.getElementById("featured-clients")) {
 
 if (document.getElementById("all-clients")) {
   const allClientsElement = document.getElementById("all-clients");
-  const allClients = [...featuredClients, ...otherClients];
+  const allClients = otherClients;
```
Comment (cehali) [inline src/js/clients.js]: I think this would remove all featured clients

File: public/audits/index.html [hunk 2]
```diff
@@ -624,7 +624,6 @@ <h5 class="mb-0 mb-1 text-danger">ABDK Consulting</h5>
 ></script>
 
 <script async src="/assets/js/theme.js"></script>
-<script src="/assets/js/add-trailing-slash.js"></script>
```
Comment (cehali) [inline public/audits/index.html:627]: Is it ok that we remove it in many files?

**Review by cehali — CHANGES_REQUESTED** [review]
Two minor comments

File: public/audits/index.html (hunk 2 above)
Comment (cehali) [reply public/audits/index.html:627]: I see that we remove file so I assume this is ok

Comment (roszpun) [reply public/audits/index.html:627]: Yes, this script is not necessary.

File: src/js/clients.js (hunk 1 above)
Comment (roszpun) [reply src/js/clients.js]: Yes, but I don't really understand why we want to display firstly featured clients and then the same clients in the our clients section. So all of the featured clients are displayed twice.

**Review by cehali — APPROVED** [review]

**Pull request merged**